en/editor/publish/publish-cocos-play.md

```diff
@@ -18,39 +18,39 @@
 
 The specific filling rules for the relevant parameter configuration are as follows:
 
-  - **tinyPacketMode**
+  1. **tinyPacketMode**
```

Comment:
The changes to this piece suggest reverting to the original format.

Reply:
can you explain why? I don't see what you mean clearly. thank you.

Reply:
The paragraph structure was correct before the revision, but wrong after the revision.

Reply:
But before I don't think the steps aligned up. I'll take another look at it.